wafer/talks/views.py

```diff
@@ -104,12 +109,30 @@ def get_context_data(self, **kwargs):
         context['can_edit'] = self.object.can_edit(self.request.user)
         return context
 
+    def form_valid(self, form):
+        # Is there a better way to do this?
+        # I don't want to use RevisionMiddleware, because I think we
+        # want more control over things than that provides, but
+        # this feels clunky
+        with revisions.create_revision():
+            result = super(TalkUpdate, self).form_valid(form)
+            revisions.set_user(self.request.user)
+            revisions.set_comment("Talk Modified")
```

Comment:
I'm not sure how I missed the decorator option in the reversion docs.

While this is a bit more compact, I'm mainly concerned about shoving everything into form_valid. Unfortunately, I don't see a better place for this, especially as I don't want to go the ReversionMiddlewar route.